src/common/reconciler_test.go

```diff
@@ -0,0 +1,366 @@
+package common_test
+
+import (
+	"fmt"
+	"github.com/opslevel/kubectl-opslevel/common"
+	"github.com/opslevel/opslevel-go/v2023"
+	opslevel_jq_parser "github.com/opslevel/opslevel-jq-parser/v2023"
+	"github.com/rocktavious/autopilot/v2023"
+	"testing"
+)
+
+func Test_Reconciler_Reconcile(t *testing.T) {
```

Comment:
Non-blocking

I think it's unnecessary to have underscores here to make the function's name and purpose more clear. The tests are in a file dedicated to Reconciler testing, so having it all as one word `TestReconcilerReconcile` is readable enough.